src/GNDStk/string2type/test/string2type.test.cpp

```diff
@@ -0,0 +1,9 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk convert(string,type)") {
+
```

Comment:
How do I know that `string2type` works?

Reply:
Because I wrote it? :-D

Seriously, in most instances of these stubbed-out tests, the constructs in question *are* tested, elsewhere, as a consequence of testing other things. They do work correctly. These individual unit test files should be filled in at some point, though. I can move that up the priority list.